libs/community/langchain_community/utilities/steam.py

```diff
@@ -45,7 +44,7 @@ def validate_environment(cls, values: dict) -> Any:
 
         # check if the python package is installed
         try:
-            from steam import Steam
+            from steam_web_api import Steam
```

Comment:
I do not want to make this change.

This swaps the official Steam package:
https://github.com/ValvePython/steam/tree/master

By one a non official one:
https://github.com/deivit24/python-steam-api